web/src/css/flowdetail.less

```diff
@@ -21,6 +21,17 @@
         }
     }
 
+    .close-button {
+        margin-top: 2;
```

Comment:
Some leftovers here due to this now being a button?

(I also cannot resist: Two what? Two horses?)

Reply:
Yes, to align the icon with the text of the tab